install-guides/ubuntu20.md

```diff
@@ -0,0 +1,512 @@
+
+* One time only, the storage/ directory must be linked to the application:
+```
+php artisan storage:link
+```
+![image](https://user-images.githubusercontent.com/17537000/171811430-f5957e34-26ca-4079-a95b-cff94a810780.png)
+
+* Database migrations must be run:
+```
+php artisan migrate --force
+```
+![image](https://user-images.githubusercontent.com/17537000/171811653-329242e8-d3c5-4a37-8f13-0a6fef91183b.png)
+
+* One time only, Generate location data:
+```
+php artisan import:cities
+```
+![image](https://user-images.githubusercontent.com/17537000/171811764-e2bfa6f1-9eb1-4d82-ba76-3f49082998ce.png)
+
+* One time only, Generate ActivityPub federation actor:
+```
+php artisan instance:actor
+```
+![image](https://user-images.githubusercontent.com/17537000/171811803-79fa7071-71ae-4da7-8c7e-33915d8a7d6d.png)
+
+* One time only, Generate OAuth keys:
+```
+php artisan passport:keys
+```
+![image](https://user-images.githubusercontent.com/17537000/171811832-8750edea-66c7-4905-8482-6506f935d0c4.png)
+
+* Routes should be cached whenever the source code changes or whenever you change routes:
+```
+php artisan route:cache
+```
+```
+php artisan view:cache
+```
+![image](https://user-images.githubusercontent.com/17537000/171811876-a6913ac7-e50a-45fc-a5b5-42ed4e2c200b.png)
+
+* Every time you edit your .env file, you must run this command to have the changes take effect:
+```
+php artisan config:cache
+```
+![image](https://user-images.githubusercontent.com/17537000/171812304-e00cc29b-453e-4bff-b398-ab453260bf57.png)
+
+* Laravel Horizon
+```
+php artisan horizon:install
+```
+```
+php artisan horizon:publish
+```
+![image](https://user-images.githubusercontent.com/17537000/171812367-970fea18-4150-46b2-9a3b-798cdaba464b.png)
+
+## Part 9.1 - Create an administrator account
+* Create a new admin user using php artisan
+```
+php artisan user:create
+```
+![image](https://user-images.githubusercontent.com/17537000/171835654-d6d42d7f-ca0d-43ee-a397-381a0c6d7533.png)
+
+## Part 10 - Return to the root account
+* Exit the `pixelfed` account and become root again.
+```
+exit
+```
+![image](https://user-images.githubusercontent.com/17537000/171812687-e34fd89f-6bd0-4724-aed6-ef60a7dfbdc1.png)
+
+## Part 11 - Nginx and Certbot - Install
+* Install stock nginx, Install certbot to handle the TLS certificate and enable the engine service to autostart.
+```
+apt -y install nginx certbot python3-certbot-nginx
+```
+```
+systemctl enable nginx
+```
+![image](https://user-images.githubusercontent.com/17537000/171813872-b3b2066c-fa90-4e4d-98bb-8ca3f797ffed.png)
+
+* Check the certbot timer is running (Certbot will renew TLS certs automatically)
+```
+systemctl status certbot.timer
+```
+![image](https://user-images.githubusercontent.com/17537000/171818819-231ada60-060d-4eb0-8692-e9cea151307b.png)
+
+
+## Part 11.1 - Configure certbot
+```
+rm /etc/nginx/sites-enabled/default
```

Comment:
thematically this should be uncer "nginx and certbot" and not under "configure certbot"